css/editEvents.less

```diff
@@ -0,0 +1,89 @@
+#expandCollapseAllEvents {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+/* Copied from http://jsfiddle.net/KyleMit/Wc4xt/ */
+.panel-heading a:after {
+  font-family: 'Glyphicons Halflings';
+  content: "\e114"; /* glyphicon-chevron-down */
+  float: right;
+  color: grey;
+}
+.panel-heading a.collapsed:after {
+  content: "\e080"; /* glyphicon-chevron-right */
+}
+
+/*
+ * Make collapse link fill up entire panel-title
+ * Copied from http://stackoverflow.com/a/20188613
+ */
+.panel-title a {
+  display: block;
+  padding: 10px 15px;
+  margin: -10px -15px;
+}
+
+/*
+ * Progress bars have a large bottom margin for some reason.
+ * This looks gross inside of tables.
+ * https://github.com/twbs/bootstrap/issues/3905
+ */
+.table .progress {
+  margin-bottom: 0px;
+}
+.table-condensed .progress {
+  margin-bottom: 0px;
+}
+
+@iconSize: 50px;
+@thumbnailPaddingAndBorder: 2*4px + 2*1px;
+
+.panel-title .cubing-icon {
+  position: absolute;
+  font-size: @iconSize - @thumbnailPaddingAndBorder;
+  top: 20px - @iconSize / 2;
+}
+
+.panel-title .cubing-icon + span {
+  padding-left: @iconSize + 5px;
+}
+
+.btn-unlock {
+  width: 2em;
+  text-align: center;
+}
+.btn-unlock i.fa:before {
+  content: "\f023"; /* fa-lock */
+}
+.btn-unlock:hover i.fa:before {
+  content: "\f09c"; /* fa-unlock */
+}
+
+.btn-lock {
+  width: 2em;
+  text-align: center;
+}
+.btn-lock i.fa:before {
+  content: "\f09c"; /* fa-unlock */
+}
+.btn-lock:hover i.fa:before {
+  content: "\f023"; /* fa-lock */
+}
+
+select[name="roundFormat"] {
+  width: 4em;
+}
+
+.progress-bar {
+  color: black;
+}
+
+table th.tight-cell, table td.tight-cell {
+  width: 1%;
+  white-space: nowrap;
+}
+
+a[data-toggle="popover"] {
```

Comment:
I'd probably remove the a tag from this selector, just in case we end up having other elements with that feature.